circle.yml

```diff
@@ -0,0 +1,46 @@
+machine:
+  php:
+    version: 5.5.21
+  ruby:
+    version: 2.1.3
+  environment:
+    APP_ENV: testing
+
+general:
+    artifacts:
+        - "tests/codeception/_output"
+        - "storage/logs"
+
+dependencies:
+  cache_directories:
+    - elasticsearch-1.6.0
+    - node_modules
+    - public/components
+  pre:
+    - pecl install xdebug
+    - echo "memory_limit = 1024M" > ~/.phpenv/versions/5.5.15/etc/conf.d/memory.ini
+    - if [[ ! -e elasticsearch-1.6.0 ]]; then wget https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.6.0.tar.gz && tar -xvf elasticsearch-1.6.0.tar.gz; fi
```

Comment:
It's a base file so everything not needed can be removed, but at least if it is needed, it's configured correctly